### DISCUSSION TITLE
New translation in Pangasinan

### DISCUSSION THREAD

**Discussion by lorenpajarits** [sircharlo/meeting-media-manager#2303] [Translations]
### What language do you speak?

I would like to help translate M³ into a language I speak, Pangasinan (PN/pag).

### Additional context

If it is necessary to translate in Pangasinan, I'd be willing to help

**Comment by mtdvlpr** [comment]
Hi @lorenpajarits, thank you for your willing spirit! @sircharlo can help you get started.

@sircharlo: Pangasinan needs to be added as a custom language in Crowdin, which only you can do.

**Reply by lorenpajarits** [reply]
Hello @sircharlo , just to clarify, are you referring to a company named Crowdin? if no, what is it?

**Reply by mtdvlpr** [reply]
@lorenpajarits Crowdin is the service we use to manage our translations: https://crowdin.com/project/meeting-media-manager

**Reply by lorenpajarits** [reply]
oh yeah, i’d be happy to help!

**Reply by sircharlo** [reply]
Sorry for the delay! @mtdvlpr I've created the custom language.
@lorenpajarits Let us know when you've logged in to CrowdIn, so that we can add you to the project!

**Reply by lorenpajarits** [reply]
Hello. I’ve already logged in

**Reply by sircharlo** [reply]
What's your username for CrowdIn?

**Reply by sircharlo** [reply]
Ah, got it. Accepted and added you to the project :)

**Reply by lorenpajarits** [reply]
yes, thank you!
